components/navbar.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="w-full bg-roses-dark h-28">
+    <div
+      class="mx-4 lg:container h-full lg:mx-auto flex gap-10 justify-between"
+    >
+      <a href="/">
+        <div class="relative w-32 h-32 lg:w-36 lg:h-36">
+          <div class="absolute w-full h-full p-3 bg-roses-dark">
+            <img
+              src="https://assets-cdn.sums.su/YU/website/img/Roses/Roses_mark_Navigation.png"
+              class="w-full h-full aspect-square"
+              alt="Roses Live"
+            />
+          </div>
+        </div>
+      </a>
+      <div
+        class="hidden lg:flex flex-grow max-w-3xl items-center justify-between text-white font-semibold"
+      >
+        <a>Roses Live</a>
+        <a>Fixtures</a>
+        <a>Get Involved</a>
+        <a>About</a>
+        <a>Food & Drink</a>
+        <a>Shop</a>
+        <a>Map</a>
+      </div>
+      <div class="hidden lg:flex w-48 items-center">
+        <a href="https://yorksu.org/">
+          <img
+            src="https://assets-cdn.sums.su/YU/website/img/yorksu-logo-white-full.png"
+            class="w-full h-fit"
+            alt="University of York Students' Union"
+          />
+        </a>
+      </div>
+      <div class="lg:hidden flex items-center">
+        <button
+          class="text-white"
+          aria-label="Open navigation"
+          @click="isMenuOpen = !isMenuOpen"
+        >
+          <i class="fa-solid fa-bars text-4xl"></i>
+        </button>
+      </div>
+    </div>
+    <div
+      v-if="isMenuOpen"
+      class="lg:hidden absolute top-28 w-full bg-roses-dark"
+    >
+      <hr class="border-zinc-600 mt-6 mx-6" />
+      <div class="container mx-auto flex flex-col gap-10">
+        <div class="flex flex-col items-center gap-4 pt-6 text-white">
+          <a>Roses Live</a>
+          <a>Fixtures</a>
```

Comment:
We should hide these too. Do you think we should put it in a data object like we do on our main site to make it easier to update these going forward?